footer.inc

```diff
@@ -66,69 +66,88 @@
-  <div id="modal1" class="modal modal-fixed-footer" style="border-style:solid; border-width:1px; border-color:blue; border-radius:20px;">
-    <div class="modal-content">
-      <h4 class="flow-text black-text" style="padding:10% 5% 0% 5%"><b>Subscription Cancellation</b></h4>
+  <div id="modal1" class="modal modal-fixed-footer">
+    <div class="modal-content" id="jqueryvalidation">
+     <div class="container" style="width:100%">
+     <h4 class="flow-text black-text center-align" style="padding:5% 10% 5% 10%"><b>Subscription Cancellation</b></h4>
       <p class="flow-text left-align black-text" style="padding:0% 10% 0% 10%">To cancel your subscription, please enter your email address below and click on the "Submit" button.</p>
       <p class="flow-text left-align black-text" style="padding:0% 10% 0% 10%">We are sorry to see you go, but you can rejoin the Village Fund at any time by clicking on the "Give Monthly" button on the previous screen.</p>
+    </div> 
     
-    <div class="section" id="jqueryvalidation" style="display:table; width:100%; padding:0% 10% 0% 10%">
-		<form style="width:100%" id="donateForm" method="get" action="">
-		<div class="col-project valign-wrapper" style="vertical-align: middle;">
+    <div class="container center-align" id="jqueryvalidation" style="padding:0% 5% 0% 5%;">
+		<form id="cancelForm" method="get" action="" class="center-align">
+		
+		<div class="row center-align">
+		<div class="col s12 m12 l12 center-align">
 			
-         					<div class="row" style="padding:3% 3% 3% 3%; width:100%">
-         						<div class="input-field col s12 m12 l12 center-align">
-         							<i class="material-icons prefix" style="font-size:20px;">email&nbsp;&nbsp;</i>
-          							<input placeholder="jane@gmail.com" id="email" name="email" data-error=".errorTxt1" style="font-size:20px; color:blue;" id="email" type="email">
-          							<div class="errorTxt1"></div>
+         					
+         						<div class="input-field col s12 center-align">
+          							<input placeholder="jane@gmail.com" id="cancellation_email" name="email" required data-error=".errorTxt3" style="font-size:20px; color:black;" type="email">
+          							<div class="errorTxt3" style="font-size:10px; color:red; padding:1% 0% 0% 0%"></div>
           						</div>
-							</div>
-         </div>	   
-         <div class="col-project valign-wrapper" style="vertical-align: middle; width:100%">
-        			<div class="input-field col s12 m12 l12">
-        			<button id="myBtn" class="btn-large center-align waves-effect waves-light light-blue submit" type="submit" name="action" style="width:100%;">Submit</button>
+         </div> 	
+         </div>
+         <div class="row">
+          						 <div class="input-field col s12 m6 l6">  
+                                  <input id="donationFirstName" name="firstname" placeholder="first name" type="text" required  data-error=".errorTxt1" style="font-size:20px; color:black;">
```

Comment:
Why are there name fields on the cancellation form?

Reply:
because i want to send the person a cancellation email, using their first name. i also think that having a couple extra steps to cancel isn't a bad thing.

Reply:
We can send it to them using the name in the database.  An email field and big red button should be sufficient.